man/man1/dsnode.1

```diff
@@ -0,0 +1,27 @@
+.TH dsnode 1
+
+.SH NAME
+
+dsnode - The DataStorm Node server.
+
+.SH SYNOPSIS
+
+dsnode [options]
+
+.SH DESCRIPTION
+
+The DataStorm node server allows discovering of registered nodes, and provides bridging of nodes with disabled server
+endpoints. Full documentation for DataStorm Node server is available online at:
```

Comment:
What does "provides bridging of nodes with disabled server endpoints" mean?

Reply:
> bridging: nodes with disabled server endpoints can connect to dsnode to receive data from other nodes. See [Networking](https://doc.zeroc.com/datastorm/latest/features/networking) for additional information.

From https://doc.zeroc.com/datastorm/latest/features/node-server

Reply:
I would write it as:
The DataStorm node server allows other DataStorm nodes to exchange topic discovery information without relying on UDP multicast. It also helps create deployments that span multiple IP networks.